### DISCUSSION TITLE
Explorer keeps crashing, restarting, crashing, can't use PC now

### DISCUSSION THREAD

**Discussion by peedoe** [valinet/ExplorerPatcher#1433] [General]
I don't use this PC often, but a week or 2 ago started getting exception errors just sitting idle.  Would click OK and everything seemed normal.  Now when I login, desktop comes up, crashes, restarts, crashes, etc.  So frequent, I can't get anything to start except task manager and I see Explorer starting and crashing.

How can I uninstall your app to see if that is the problem if I can't get to the settings/apps or control panel/apps to uninstall it?

TIA,
Peedoe

**Comment by pyrates999** [comment]
right here: https://github.com/valinet/ExplorerPatcher/wiki#uninstalling
